src/Form/Form.php

```diff
@@ -270,6 +270,28 @@ public function getData(?string $field = null)
         return Hash::get($this->_data, $field);
     }
 
+    /**
+     * Saves a variable or an associative array of variables for use inside form data.
+     *
+     * @param string|array $name A string or an array of data.
+     * @param mixed $value Value in case $name is a string (which then works as the key).
+     *   Unused if $name is an associative array, otherwise serves as the values to $name's keys.
+     * @return $this
+     */
+	public function set($name, $value = null) 
```

Comment:
If we're going to add a new method, could it only add a single key? We already have `setData()` to set all the keys.

Reply:
`setData()` overrides all existing data. `set()` appends new data using `Hash::insert()`.

		$form->set('var1', 'val1'); //array ( 'var1' => 'val1', )
		$form->set(['var2'=>'val2', 'var3'=>'val3']); //array ( 'var1' => 'val1', 'var2' => 'val2', 'var3' => 'val3', )
		$form->set(['var1'=>'val1_new']); //array ( 'var1' => 'val1_new', 'var2' => 'val2', 'var3' => 'val3', )
		$form->setData(['var1'=>'val1_new2']); //array ( 'var1' => 'val1_new2', )

Reply:
Ok. Are you aiming to replicate the behavior of `ViewVarsTrait::set()` on Form?

Reply:
Yes. Actually I want to replicate `EntityTrait::set()` first, and `EntityTrait::setError()` in next PR.
Both are used as forms context, but set() and setError() works for entity only.

Reply:
>  Both are used as forms context, but set() and setError() works for entity only.

I don't see `set()` and `setError()` being used in the `FormContext`?

Reply:
I mean that entities and forms are used as form context. Not methods.